site/topics/programming/assembly.rst

```diff
@@ -22,6 +22,53 @@ Assembly Language
 Assembler
 =========
 
+* An assembler is a tool to convert assembly language to machine code
+* An assembly language is a very low level programming language
+
+    * Often referred to as *assembly*
```

Comment:
This point feels weird. _Assembly_ is mentioned twice, then it's like "btw the language is called assembly."

Reply:
I'll re order the things as I think that helps